<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeMarker</class>
 <widget class="QWidget" name="CodeMarker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>637</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>dictionary: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dictList">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="showDetectedCandidates">
       <property name="text">
        <string>show detected candidates</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>detection parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="maxMarkerPerimeter">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="prefix">
         <string>max: </string>
        </property>
        <property name="suffix">
         <string> cm</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>15.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="minMarkerPerimeter">
        <property name="prefix">
         <string>min: </string>
        </property>
        <property name="suffix">
         <string> cm</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="adaptiveThreshWinSizeMax">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="prefix">
         <string>max: </string>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="value">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>marker border size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="adaptiveThreshConstant">
        <property name="prefix">
         <string>constant: </string>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="markerBorderBits">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>adaptive thresholding window:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="adaptiveThreshWinSizeStep">
        <property name="prefix">
         <string>step: </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="polygonalApproxAccuracyRate">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.030000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>max ratio error:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="maxErroneousBitsInBorderRate">
        <property name="prefix">
         <string>max: </string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.350000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>marker perimeter:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="minCornerDistance">
        <property name="prefix">
         <string>min: </string>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="adaptiveThreshWinSizeMin">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="prefix">
         <string>min: </string>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>marker distance:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="minDistanceToBorder">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="prefix">
         <string>min: </string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>corner distance:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>error bits in border rate:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>error correction rate:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>distance to border:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="minMarkerDistance">
        <property name="prefix">
         <string>min: </string>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QDoubleSpinBox" name="errorCorrectionRate">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.600000000000000</double>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>otsu std. dev.: </string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QDoubleSpinBox" name="minOtsuStdDev">
        <property name="prefix">
         <string>min: </string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>perspective remove</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QSpinBox" name="perspectiveRemovePixelPerCell">
           <property name="suffix">
            <string> px</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>per cell:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>ignored margin per cell:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="perspectiveRemoveIgnoredMarginPerCell">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.130000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QGroupBox" name="doCornerRefinement">
        <property name="title">
         <string>use subpixel refinement</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QSpinBox" name="cornerRefinementWinSize">
           <property name="suffix">
            <string> px</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>max iterations:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="cornerRefinementMaxIterations">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>min accuracy:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="cornerRefinementMinAccuracy">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_10">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>win size:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QPushButton" name="moreInfosButton">
        <property name="text">
         <string>More infos...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
